state-chain/chains/src/dot.rs

```diff
@@ -252,6 +252,76 @@ impl Default for PolkadotTrackedData {
 	}
 }
 
+/// See https://wiki.polkadot.network/docs/learn-transaction-fees
+mod fee_constants {
+	use frame_support::sp_runtime::FixedU128;
+
+	// See https://wiki.polkadot.network/docs/learn-DOT.
+	pub const MICRO_DOT: u128 = 10_000;
+	pub const MILLI_DOT: u128 = 1_000 * MICRO_DOT;
+
+	/// Taken from the Polkadot runtime.
+	pub const MULTIPLIER: FixedU128 = FixedU128::from_inner(100_000_000_000_000_000);
```

Comment:
Isnt this a variable? This would cary according to the traffic on polkadot chain?

Reply:
It might be but we don't track it and I don't want to bloat this PR. It also seems to be static right now on Polkadot.